resources/modernizer_exclusions.txt

```diff
@@ -20,3 +20,4 @@ com/google/common/primitives/Longs.compare:(JJ)I
 com/google/inject/Inject
 com/google/inject/Provider
 java/lang/StringBuffer."<init>":()V
+java/util/Hashtable."<init>":(I)V
```

Comment:
Unfortunately no :( The modernizer plugin just reads the exclusions but you cannot configure them "just for one package or class".
We could override the configuration just for the scriptbuilder project, but if at some point we need to use the registerService in core, we'll have the same issue. I think the exclusion makes sense there, globally, to ensure we can use the OSGi 6 framework method signatures where needed.